setup.py

```diff
@@ -29,6 +29,7 @@
 QUALITY_REQUIRES = [
     "black",
     "ruff",
+    "isort",
```

Comment:
Not needed no?

Reply:
we do need it see: https://github.com/huggingface/optimum-neuron/blob/6a340cd324526b511200c3fef07a68f56abfa63f/pyproject.toml#L29
ruff is not installing it by default. You might have it already installed in your env but that was a issue on my side.